docs/source/libraries/keras.rst

```diff
@@ -0,0 +1,81 @@
+.. _library-keras:
+
+Keras
+=====
+
+Keras_ is "a high-level neural networks API, written in Python and capable of running on top of TensorFlow, CNTK, or Theano". 
+
+Keras can be used for many Machine Learning tasks, and it has support for both popular
+and experimental neural network architectures.
+
+.. _Keras: https://keras.io/
+
+
+.. _keras-explain-prediction:
+
+explain_prediction
+------------------
+
+Currently ELI5 supports :func:`eli5.explain_prediction` for Keras image classifiers.
+:func:`eli5.explain_prediction` explains image classifications through `Grad-CAM <https://arxiv.org/pdf/1610.02391.pdf>`_.
+
+The returned :class:`eli5.base.Explanation` object contains some important objects:
+
+* ``image`` represents the image that is inputted into the model.
+
+* ``heatmap``  (property of each :class:`eli5.base.TargetExplanation` in ``targets``) is a grayscale "localization map", roughly indicating regions of importance in the image for the predicted class.
```

Comment:
Do we want to be more specific here, e.g. 

* what grayscale means? Is it a 2d array with numbers from 0 to 255, with floats from 0 to 1, with arbitrary floats, including negative? 
* what do these numbers represent - e.g. 
  * what do negative numbers mean for a particular target,
  * what does it mean if model predicts that the object is not of a particular class, but some numbers in heatmap are high - does it mean object is not of this class *because* of these regions, or does it mean that regions represent some "evidence" that the object belongs to a class, but evidence is not enough?

I think we may be able to explain this all here, because a specific algorithm (Grad-CAM) is mentioned above.

Reply:
Sensible changes, especially about explaining what Grad-CAM actually does. I didn't want to include specific details about the ``heatmap``, etc as I thought it would change later, but it's fine for the PR I guess.

https://github.com/TeamHG-Memex/eli5/pull/315/commits/d7aae228da838afc0fea6133a6d19c9934c1d73e

Reply:
I think specific details are fine here, because these are docs for Keras models explained by Grad-CAM. You're right that .heatmap may meean something else in a different context, so probbaly we shouldn't explain it like this in some general docs about TargetExplanation data structure.

Reply:
Removed mention of TargetExplanation in `libraries/keras.rst`.

https://github.com/TeamHG-Memex/eli5/pull/315/commits/359aed8c1c7852bd8388720674941a6262bf0f5b